src/domain-registry-essentials/cns-smart-contracts.md

```diff
@@ -125,167 +125,166 @@ URIPrefixController enables the Unstoppable Domains team to edit the token URI p
 
-IRegistry interface declares all the Registry events and methods \(both read and write ones\).
+The `IRegistry` interface declares all the `Registry` events and methods (both read and write).
 
 **Source code:** [contracts/IRegistry.sol](https://github.com/unstoppabledomains/dot-crypto/blob/master/contracts/IRegistry.sol)
 
 **Implemented by:**
 
-* [Registry](cns-smart-contracts.md#registry)
+- [Registry](cns-smart-contracts.md#registry)
 
 ### IRegistryReader
 
-IRegistryReader interface declares only read-only Registry methods.
+The `IRegistryReader` interface declares only read-only `Registry` methods.
 
 **Source code:** [contracts/IRegistryReader.sol](https://github.com/unstoppabledomains/dot-crypto/blob/master/contracts/IRegistryReader.sol)
 
 **Implemented by:**
 
-* [Registry](cns-smart-contracts.md#registry)
-* [ProxyReader](cns-smart-contracts.md#proxyreader)
+- [Registry](cns-smart-contracts.md#registry)
+- [ProxyReader](cns-smart-contracts.md#proxyreader)
 
 ### IResolver
 
-IResolver interface declares the minimal set of Resolver methods that are used for configuring domain records.
+The `IResolver` interface declares the minimal set of `Resolver` methods for configuring domain records.
 
 **Source code:** [contracts/IResolver.sol](https://github.com/unstoppabledomains/dot-crypto/blob/master/contracts/IResolver.sol)
 
 **Implemented by:**
 
-* [Resolver](cns-smart-contracts.md#resolver)
+- [Resolver](cns-smart-contracts.md#resolver)
 
 ### IResolverReader
 
-IResolverReader interface declares the set of Resolver methods that can are used for reading Resolver records.
+The `IResolverReader` interface declares the set of methods for reading `Resolver` records.
 
 **Source code:** [contracts/IResolverReader.sol](https://github.com/unstoppabledomains/dot-crypto/blob/master/contracts/IResolverReader.sol)
 
 **Implemented by:**
 
-* [Resolver](cns-smart-contracts.md#resolver)
-* [ProxyReader](cns-smart-contracts.md#proxyreader)
+- [Resolver](cns-smart-contracts.md#resolver)
+- [ProxyReader](cns-smart-contracts.md#proxyreader)
 
 ### IDataReader
 
-IDataReader interface declares the methods that are unique to the ProxyReader smart contract, which returns combined data from the Registry and Resolver contracts.
+The `IDataReader` interface declares the methods that are unique to the `ProxyReader` smart contract, which returns combined data from the `Registry` and `Resolver` contracts.
 
 **Source code:** [contracts/IDataReader.sol](https://github.com/unstoppabledomains/dot-crypto/blob/master/contracts/IDataReader.sol)
 
 **Implemented by:**
 
-* [ProxyReader](cns-smart-contracts.md#proxyreader)
+- [ProxyReader](cns-smart-contracts.md#proxyreader)
 
 ### IMintingController
 
-IMintingController interface declares a set of methods for minting, that both MintingController and WhitelistedMinter implement.
+The `IMintingController` interface declares a set of methods for minting which both `MintingController` and `WhitelistedMinter` implement.
```

Comment:
> which both `MintingController` and `WhitelistedMinter` implement

It doesn't sound like a restrictive clause to me (I believe we still need a comma there).